Simplified/NYPLBookAcquisitionPath.m

```diff
@@ -42,7 +41,6 @@ - (instancetype _Nonnull)initWithRelation:(NYPLOPDSAcquisitionRelation const)rel
       ContentTypeOpenAccessAudiobook,
       ContentTypeOpenAccessPDF,
       ContentTypeFeedbooksAudiobook,
-      ContentTypeOverdriveCatalog,
```

Comment:
can you explain why we should remove this? wouldn't this remove OD audiobooks from the visible catalog?

Reply:
Just found out the content of OverdriveCatalog is same as the opds catalog, so this does nothing.
Probably OD does not have its own catalog acquisition and I made a mistake at the very beginning.